### ISSUE TITLE
Improvements on Undo/Redo

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#91]
Related to #76:

- [ ] Consider compound edition so undo undoes words or lines, not only char by char.
- [x] Make sure this allows coming back to the status of the area before the lemmatisation is invoked.
- [x] Assign key bindings to undo/redo.

**Comment by raquel-ucl** [comment]
Key bindings for undo/redo added in #162. 
Undo lemmatisation works now after #167.
The other part of this issue is low priority for now, so I'll move it to a new issue.
